contracts/zero-ex/contracts/src/features/interfaces/IOtcOrdersFeature.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2021 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6.5;
+pragma experimental ABIEncoderV2;
+
+import "../libs/LibNativeOrder.sol";
+import "../libs/LibSignature.sol";
+
+
+/// @dev Feature for interacting with OTC orders.
+interface IOtcOrdersFeature {
+
+    /// @dev Options for handling ETH/WETH conversion
+    /// @param LeaveAsWeth Neither unwrap nor wrap.
+    /// @param WrapEth Wrap attached ETH.
+    /// @param UnwrapWeth Unwrap WETH before transferring
+    ///        to taker.
+    enum WethOptions {
+        LeaveAsWeth,
+        WrapEth,
```

Comment:
During the fill of a quote, should we have error checks such as

```
- if not selling ETH, raise an error if WrapEth is set
- if not buy ETH, raise an error if UnwrapWeth is set

Reply:
we do